SwiftWisdomTests/StandardLibrary/DictionaryTests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DictionaryTests.swift
+//  SwiftWisdom
+//
+//  Created by Alexander Persian on 9/30/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+import XCTest
+
+class DictionaryTests: XCTestCase {
+
+    func testSetValueForKeypath() {
+        var testDictionary: Dictionary<String, Int> = [:]
+        testDictionary.ip_set(value: 42 as AnyObject, forKeypath: "hello")
+        XCTAssertEqual(testDictionary["hello"], 42)
+    }
+
+    func testValueForKeypath() {
+        let testDictionary: Dictionary<String, Int> = ["hello" : 42]
```

Comment:
for keypath test, it would be good to test nested dictionaries(since that is what the keypath is for), so have a dictionary of nested dictionaries - dont need to declare the type since swift will do that for you

```
let testDictionary = ["hello" : ["world : 42]]
testDictionary.ip_value(forKeypath: "hello.world")
```